docs/api-reference/test-utils/browser-test-driver.md

```diff
@@ -98,3 +98,26 @@ window.browserTest('done', 'custom message');
 
 Notify the node script that the app has finished executing and the browser should be closed.
 
+### image-diff
+
+```js
+window.browserTest('image-diff', options);
```

Comment:
I wonder if a sendMessages type function returning overloaded types (`Promise<object>` in this case) is the best design, compared to an explicit API:

```
window.browserTestDriver.captureAndDiffScreen(options);
```

If message is preferred, use verb-noun? `image-diff` => `diff-image`?